builtInFunctions/logsAndEvents.go

```diff
@@ -79,18 +79,29 @@ func newEntryForESDT(identifier, tokenID []byte, nonce uint64, value *big.Int, a
 	return logEntry
 }
 
-func extractTokenIdentifierAndNonceESDTWipe(args []byte) ([]byte, uint64) {
+func extractTokenIdentifierAndNonceESDTWipe(esdtPrefix []byte, args []byte) ([]byte, uint64) {
 	argsSplit := bytes.Split(args, []byte(esdtIdentifierSeparator))
-	if len(argsSplit) < 2 {
+	if len(argsSplit) < 2 ||
+		len(argsSplit) < 3 && len(esdtPrefix) != 0 {
```

Comment:
I would suggest we extract `len(argsSplit) < 3 && len(esdtPrefix) != 0` to a bool variable